src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -68,8 +71,19 @@ export const getUploadPhotosFormInitialValues = (
           url: getPhotoUrlFromAsset(asset),
           removed: false,
           loading: false,
-        })) || [],
+        })) || []
+  } else if (myCollectionArtwork) {
+    photos =
+      myCollectionArtwork?.images
+        ?.map(image => ({
+          name: "Automatically added",
+          externalUrl: image?.url!,
+          type: "image/jpg",
```

Comment:
**thought:** do we only allow jpg images?

Reply:
The image from the artwork comes as a jpg. If the user uploads a file manually, we support more file types and take the type directly from the uploaded file.